### PR TITLE
Add auto-remediation guide for Sensu Core 2.0

### PR DESCRIPTION
**Pull request by palourde**
Closes https://github.com/sensu/sensu-docs-site/issues/276

### CODE REVIEW

File: content/sensu-core/2.0/guides/run-auto-remediation.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Validating the check hook
+
+You can verify the proper behavior of the check hook against a specific event by
+using `sensuctl`. It might take a few moments, once the check hook is assigned,
+for the check to be scheduled on the entity and the result sent back to Sensu
+backend. The check hook command result will be available in the `hooks` array,
+within the `check` hash.
+
+{{< highlight shell >}}
+sensuctl event info i-424242 nginx_process --format json
```
Comment (mercul3s) [inline content/sensu-core/2.0/guides/run-auto-remediation.md]: I'd suggest adding an example of the hook output here.

**Review by mercul3s — COMMENTED** [review]
One small note but otherwise 🍕 🌮 🍨 🍔 🌯

File: content/sensu-core/2.0/guides/run-auto-remediation.md (hunk 1 above)
Comment (palourde) [reply content/sensu-core/2.0/guides/run-auto-remediation.md]: Done!

**Review by mercul3s — APPROVED** [review]

**Pull request merged**